layers/timeline_semaphore.c

```diff
@@ -2183,6 +2186,26 @@ static void timeline_DestroyInstance(
     struct instance_data *instance = object_find(&global_map, _instance);
     PFN_vkDestroyInstance destroy_cb = instance->vtable.DestroyInstance;
 
+    // Unmap physical devices from global_map
+    VkPhysicalDevice* pPhysicalDevices = vk_alloc(&instance->alloc,
+            sizeof(VkPhysicalDevice*) * instance->physicalDeviceCount,
+            8, VK_SYSTEM_ALLOCATION_SCOPE_COMMAND);
+
+    VkResult result = instance->vtable.EnumeratePhysicalDevices(_instance,
+                                                                &instance->physicalDeviceCount,
+                                                                pPhysicalDevices);
+
+    if (result == VK_SUCCESS || result == VK_INCOMPLETE) {
+        for (uint32_t i = 0; i < instance->physicalDeviceCount; i++) {
+            if (object_find(&global_map, pPhysicalDevices[i]))
```

Comment:
We probably don't need to check it's in the map, just remove everything :)

`object_unmap` should deal with entries not present.

Reply:
ok